ros_ws/src/crazyswarm/src/crazyswarm_server.cpp

```diff
@@ -1334,9 +1276,8 @@ class CrazyflieGroup
   std::string m_interactiveObject;
   libobjecttracker::ObjectTracker* m_tracker;
   int m_radio;
-  // ViconDataStreamSDK::CPP::Client* m_pClient;
   pcl::PointCloud<pcl::PointXYZ>::Ptr m_pMarkers;
-  std::vector<libmotioncapture::Object>* m_pMocapObjects;
+  std::map<std::string, libmotioncapture::RigidBody>* m_pMocapRigidBodies;
```

Comment:
If I understand correctly, we can't make `m_pMocapRigidBodies` a reference because it might be null if we are using libobjecttracker and no interactive object. If correct, could add a comment that this is a non-owning pointer.